pages/docs/user-docs/docs-bootstrap.md

```diff
@@ -48,7 +129,7 @@ The Docker bootstrap module will create a core operating system image based on a
 Once the `Bootstrap` module has completed, the sections are identified and utilized if present. The following sections are supported in the bootstrap definition, and integrated during the bootstrap process:
 
 
-#### %setup
+### %setup
 This section blob is a Bourne shell scriptlet which will be executed on the host outside the container during bootstrap. The path to the container is accessible from within the running scriptlet environment via the variable `$SINGULARITY_ROOTFS`. For example, consider the following scriptlet:
```

Comment:
we will need to add %help to the development version (2.4) docs! Can you PR these general changes there too? It will make the merge (when we get there) easier.